src/screens/playing.rs

```diff
@@ -3,9 +3,8 @@
 use bevy::{input::common_conditions::input_just_pressed, prelude::*};
 
 use super::Screen;
-use crate::game::{
-    assets::SoundtrackHandles, audio::soundtrack::SoundtrackCommands as _, spawn::level::SpawnLevel,
-};
+use crate::demo::level::SpawnLevel;
+use crate::{assets::SoundtrackHandles, audio::soundtrack::SoundtrackCommands as _};
```

Comment:
I like 1 line per import personally, but should merge these to stay consistent

Reply:
Ran nightly cargo fmt.